docs/src/api/class-cdpsession.md

```diff
@@ -25,23 +25,23 @@ await client.send('Animation.setPlaybackRate', {
 
 ```python async
 client = await page.context().new_cdp_session(page)
-await client.send("animation.enable")
-client.on("animation.animation_created", lambda: print("animation created!"))
-response = await client.send("animation.get_playback_rate")
-print("playback rate is " + response["playback_rate"])
-await client.send("animation.set_playback_rate", {
-    playback_rate: response["playback_rate"] / 2
+await client.send("Animation.enable")
+client.on("Animation.animationCreated", lambda: print("animation created!"))
+response = await client.send("Animation.getPlaybackRate")
+print("playback rate is " + response["playbackRate"])
```

Comment:
Since `response["playbackRate"]` is int, str + int throws an error.

`f'playback rate is {esponse["playbackRate"]}'` or `"playback rate is " + str(response["playbackRate"])` is working